### ISSUE TITLE
Training speed will be slow down after some iters

### ISSUE THREAD

**Issue by Hou9612** [megvii-model/ShuffleNet-Series#20]
Thanks for your code. When I train Shufflenetv2 1.0 , The training speed will be slow after some iters. As you can see in the picture, at the beginning 1000 iters, I need 4s to train 20 iters, but after this time, I need about 16s to train 20iters. I use 3 titan xp to train and use NVIDIA-DALI to accelerate dataloader. I set batch size as 512. Has anyone missed the same situation as me? 
![111](https://user-images.githubusercontent.com/49936825/67290278-7b65db00-f512-11e9-9d07-2a10eec684ad.png)
![222](https://user-images.githubusercontent.com/49936825/67290295-7ef96200-f512-11e9-9d80-6c4c46adaaef.png)

**Comment by nmaac** [comment]
No we haven't met this problem. Maybe it is caused by your experimental environment.
